Gemfile

```diff
@@ -11,32 +11,43 @@ gem 'sqlite3', '~> 1.4'
 gem 'puma', '~> 3.11'
 # Use SCSS for stylesheets
 gem 'sass-rails', '~> 5'
+
+# Use the normal sprockets way for compiling assets
+gem 'sprockets'
```

Comment:
I rearranged some definitions to make more sense, but this is the key line. We didn't have this library installed